packages/checkout/widgets-lib/src/context/view-context/SaleViewContextTypes.ts

```diff
@@ -52,3 +58,8 @@ export enum FundWithSmartCheckoutSubViews {
   FUNDING_ROUTE_SELECT = 'FUNDING_ROUTE_SELECT',
   FUNDING_ROUTE_EXECUTE = 'FUNDING_ROUTE_EXECUTE',
 }
+
+export enum OrderSummarySubViews {
```

Comment:
This is a small nit.. but wondering if its clearer to take our `sub` ? so just `OrderSummaryViews`?
```suggestion
export enum OrderSummaryViews {
```